test/unit/model/registerConsent.model.test.ts

```diff
@@ -58,89 +62,98 @@ jest.mock('~/shared/pub-sub')
 
 jest.mock('axios')
 
+// mocking the createAndStoreConsent function since
+// our test payload does not have any scopes so the function will fail
+// todo: un-mock this once we have a better payload
+jest.mock('~/domain/consents')
+
+// todo: update this test data with a payload that includes scope actions
 const consentsPostRequestAUTH: tpAPI.Schemas.ConsentsPostRequestAUTH = {
-  consentId: 'b51ec534-ee48-4575-b6a9-ead2955b8069',
+  consentId: 'ee54e4b1-180e-40a5-b9fd-d44b62587b89',
   scopes: [
+    // @ts-ignore
     {
-      accountId: 'dfsp.username.5678',
-      actions: [
-        'accounts.transfer',
-        'accounts.getBalance'
-      ]
-    }
+      accountId: 'f67170bc-3702-4786-90cd-023466b575ca',
+      // TODO: missing in payload for some reason on front end client...
+      // This isn't a big deal, as long as we derive the challenge in the same way.
+      // actions: [
```

Comment:
Is this fixed now with the newer payload I gave you?

Reply:
yep 🙃 I'll remove this comment in a bit

Reply:
Cool